packages/fast-components/src/progress/base-progress.ts

```diff
@@ -0,0 +1,18 @@
+import { attr, FastElement } from "@microsoft/fast-element";
+
+export class BaseProgress extends FastElement {
+    @attr
+    public value: number;
```

Comment:
Before we merge, we could use the new attr converter feature to set these up with a number converter, ensuring that the values are always numbers (and not strings) when coming from HTML.